src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -898,30 +900,31 @@ def merge_units(
         new_unit_ids : None or list
             A new unit_ids for merged units. If given, it needs to have the same length as `merge_unit_groups`. If None,
             merged units will have the first unit_id of every lists of merges
-        censor_ms : None or float
-            When merging units, any spikes violating this refractory period will be discarded. Default is None
-        merging_mode : "soft" can be in ["soft", "hard"]
-            How merges are performed. In the "soft" mode, merges will be approximated, with no reloading of the
-            waveforms. This will lead to approximations. If "hard", recomputations are accuratly performed,
+        censor_ms : None or float, default: None
+            When merging units, any spikes violating this refractory period will be discarded. If None all units are kept
+        merging_mode : ["soft", "hard"], default: "soft"
+            How merges are performed. If the `merge_mode` is "soft" , merges will be approximated, with no reloading of the
+            waveforms. This will lead to approximations. If `merge_mode` is "hard", recomputations are accurately performed,
             reloading waveforms if needed
         sparsity_overlap : float, default 0.75
             The percentage of overlap that units should share in order to accept merges. If this criteria is not
-            achieved, soft merging will not be possible
+            achieved, soft merging will not be possible and an error will be raised
         new_id_strategy : "append" | "take_first", default: "append"
             The strategy that should be used, if `new_unit_ids` is None, to create new unit_ids.
-                * "append" : new_units_ids will be added at the end of max(sorging.unit_ids)
+                * "append" : new_units_ids will be added at the end of max(sorting.unit_ids)
                 * "take_first" : new_unit_ids will be the first unit_id of every list of merges
-        folder : Path or None
-            The new folder where selected waveforms are copied
-        format : "auto" | "binary_folder" | "zarr"
-            The format of the folder.
-        verbose:
+        folder : Path or None | default: None
+            The new folder where selected waveforms are copied for `format` "binary_folder" or "zarr"
+        format : "memory" | "binary_folder" | "zarr", default: "memory"
+            The format of SortingAnalyzer
+        verbose : bool, default: False
+            Whether to display calculations (such as sparsity estimation)
 
 
         Returns
         -------
         analyzer :  SortingAnalyzer
-            The newly create sorting_analyzer with the selected units
+            The newly create SortingAnalyzer with the selected units
```

Comment:
```suggestion
            The newly create `SortingAnalyzer` with the selected units
```
Did we agree this should have ticks?